### PR TITLE
Adding script to pull concept IDs related to metformin and its brand …

### PR DESCRIPTION
**Pull request by finnkuusisto**
New script pulls metformin concepts, including name brands. I'm seeing 14,096 hits in the current database. I manually reviewed a sample of the rows, and didn't find any problems.

### CODE REVIEW

**Comment by afbarnard** [comment]
Did you check the `domain_id`s to see if there was anything weird? (`cut -d '|' -f 3 <csv> | sort | uniq -c`)

The code seems fine.

**Comment by finnkuusisto** [comment]
I hadn't previously, but it's all 'Drug.'

File: src.bash/concept.metformin.raw.csv.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    concept_name like '%glumetza%' or
+    concept_name like '%obimet%' or
+    concept_name like '%gluformin%' or
+    concept_name like '%dianben%' or
+    concept_name like '%diabex%' or
+    concept_name like '%diaformin%' or
+    concept_name like '%siofor%' or
+    concept_name like '%metfogamma%' or
+    concept_name like '%gilfor%'
+  )
```
Comment (afbarnard) [inline src.bash/concept.metformin.raw.csv.sh:40]: I found a false positive, cobimetinib, and so would recommend adding the following code:
```suggestion
  ) and (
    -- Exclude major categories of false positives
    concept_name not like '%cobimetinib%'
  )
```

**Review by afbarnard — CHANGES_REQUESTED** [review]
Here's how I found the false positive. I ran `bash concept.metformin.raw.csv.sh <vcb.sqlite> | grep -iv -e metformin | less` and looked through the output (only about 300 rows). For each name that's a metformin synonym, I added it to the list of patterns for grep. I ended up with `bash concepts.metformin.raw.csv.sh <vcb.sqlite> | grep -iv -e metformin -e diabex -e diaformin -e glucophage -e glumetza -e riomet -e fortamet -e siofor -e metfogamma | less`. Excluding those synonyms and updating the query to exclude cobimetinib left me with an empty file; nothing more to check. So, after removing that one false positive, everything seems fine. (Granted, this process is not foolproof, but probably good enough and handles many more rows than inspectable by hand.)

**Comment by finnkuusisto** [comment]
Updated. I didn't see any other false positives using your grep approach.

**Review by afbarnard — APPROVED** [review]

**Pull request merged**